cartography/intel/aws/ecr.py

```diff
@@ -148,9 +151,12 @@ def sync(
         logger.info("Syncing ECR for region '%s' in account '%s'.", region, current_aws_account_id)
         image_data = {}
         repositories = get_ecr_repositories(boto3_session, region)
-        for repo in repositories:
-            repo_image_obj = get_ecr_repository_images(boto3_session, region, repo['repositoryName'])
+
+        async def async_get_images(repo: Dict[str, Any]) -> None:
```

Comment:
For someone like me who does not know how `async` works, can you briefly describe what's going on here? E.g. how many calls to `get_ecr_repository_images` are happening in parallel, is it configurable, etc?

Reply:
changed